### PR TITLE
 Authorization/Authentication added

### PR DESCRIPTION
**Pull request by Xakimov1610**
1- Program.cs ga   `app.UseAuthentication();` ni Registratsiya qildim ,

2- `ConfigureApplicationCookie` ni ham

### CODE REVIEW

File: webapp/Program.cs [hunk 1]
```diff
@@ -3,6 +3,17 @@
 // Add services to the container.
 builder.Services.AddControllersWithViews();
 
+builder.Services.ConfigureApplicationCookie(options =>
+{
+    options.Cookie.Name = "register.cookie";
+    options.LoginPath = "/account/signin";
+    options.LogoutPath = "/account/signout";
```
Comment (wahid-d) [inline webapp/Program.cs:10]: Shu ikki qator kerak emas

File: webapp/Program.cs [hunk 2]
```diff
@@ -3,6 +3,17 @@
 // Add services to the container.
 builder.Services.AddControllersWithViews();
 
+builder.Services.ConfigureApplicationCookie(options =>
+{
+    options.Cookie.Name = "register.cookie";
```
Comment (wahid-d) [inline webapp/Program.cs:8]: Cookie name = "invoice.app.identity"

File: webapp/Program.cs [hunk 3]
```diff
@@ -3,6 +3,17 @@
 // Add services to the container.
 builder.Services.AddControllersWithViews();
 
+builder.Services.ConfigureApplicationCookie(options =>
+{
+    options.Cookie.Name = "register.cookie";
+    options.LoginPath = "/account/signin";
+    options.LogoutPath = "/account/signout";
+
+    // options.Cookie.Expiration = TimeSpan.FromSeconds(10);
```
Comment (wahid-d) [inline webapp/Program.cs:12]: Remove comment

File: webapp/Program.cs [hunk 4]
```diff
@@ -3,6 +3,17 @@
...
 builder.Services.AddControllersWithViews();
 
+builder.Services.ConfigureApplicationCookie(options =>
+{
+    options.Cookie.Name = "register.cookie";
+    options.LoginPath = "/account/signin";
+    options.LogoutPath = "/account/signout";
+
+    // options.Cookie.Expiration = TimeSpan.FromSeconds(10);
+    options.Cookie.MaxAge = TimeSpan.FromDays(15);
```
Comment (wahid-d) [inline webapp/Program.cs:13]: MaxAge exception beradi.Uni udalit qiling. Orniga Expiratoin ishlating

**Review by wahid-d — CHANGES_REQUESTED** [review]

File: webapp/Program.cs (hunk 4 above)
Comment (Xakimov1610) [reply webapp/Program.cs]: @wahid-d O'zgartirildi

**Comment by Xakimov1610** [comment]
resolve qildim @wahid-d

File: webapp/Program.cs [hunk 5]
```diff
@@ -3,6 +3,14 @@
 // Add services to the container.
 builder.Services.AddControllersWithViews();
 
+builder.Services.ConfigureApplicationCookie(options =>
+{
+    options.Cookie.Name = "invoice.app.identity";
+
```
Comment (wahid-d) [inline webapp/Program.cs:9]: ortiqcha qator

**Review by wahid-d — CHANGES_REQUESTED** [review]

**Pull request merged**